.github/styles/vocab.txt

```diff
@@ -75,6 +75,7 @@ Firekube
 Fiverr
 freben
 Fredrik
+gitbeaker
```

Comment:
Prolly just make it a code block to skip this? 😅

Reply:
Eh, I was glad to have it in here anyway.